sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

```diff
@@ -488,21 +492,21 @@ static void send_serial_number(void)
       publish_message.topic, publish_message.out_payload, IOT_SAMPLE_MQTT_PUBLISH_QOS);
   IOT_SAMPLE_LOG_SUCCESS(
       "Client sent `%.*s` reported property message.",
-      az_span_size(twin_reported_serial_number_property_name),
-      az_span_ptr(twin_reported_serial_number_property_name));
+      az_span_size(property_reported_serial_number_property_name),
+      az_span_ptr(property_reported_serial_number_property_name));
   IOT_SAMPLE_LOG_AZ_SPAN("Payload:", publish_message.out_payload);
   IOT_SAMPLE_LOG(" "); // Formatting
 
   // Receive the response from the server.
   receive_mqtt_message();
 }
 
-static void request_device_twin_document(void)
+static void request_device_property_document(void)
 {
-  IOT_SAMPLE_LOG("Client requesting device twin document from service.");
+  IOT_SAMPLE_LOG("Client requesting device property document from service.");
 
-  // Get the Twin Document topic to publish the twin document request.
-  az_result rc = az_iot_pnp_client_twin_document_get_publish_topic(
+  // Get the property document topic to publish the property document request.
+  az_result rc = az_iot_pnp_client_property_document_get_publish_topic(
```

Comment:
I hadn't noticed `property_document` in api name but my comment above RE this being non-standard would apply here too.

Reply:
Yea this is symmetrical with the Hub API. We should probably discuss whether we want to deviate from that.